Resources/Prototypes/_Sunrise/InventoryTemplates/penguin_inventory_template.yml

```
@@ -0,0 +1,13 @@
+- type: inventoryTemplate
+  id: penguin
+  slots:
+  - name: mask
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)